src/planning.py

```diff
@@ -231,6 +240,8 @@ def _skeleton_generator(
     """
 
     start_time = time.time()
+    num_rollout_steps = CFG.pg3_max_policy_guided_rollout
```

Comment:
this is now a general function that is not necessarily going to use pg3. so let's add another hyperparameter in settings called something like `sesame_max_policy_guided_rollout` and use that instead. we can keep the value at 50.